source/_integrations/vesync.markdown

```diff
@@ -111,6 +121,44 @@ VeSync air purifiers will expose the following details depending on the features
 | `night_light`           | The current status of the night light (Core200S/Core400s)                         | off             |
 | `child_lock`            | The current status of the child lock (Core200S/300s/400s)                         | off             |
 
+## Humidity Sensors
+
+VeSync humidifiers include humidity sensors. The measured humidity is exposed as a diagnostic sensor entity alongside the
+humidifier itself.
+
+| Sensor                                      | Description                                     | Example |
+| ------------------------------------------- | ----------------------------------------------- | ------- |
+| `sensor.<humidifier name>_current_humidity` | The percent humidity measured by the humidifier | 43      |
+
+## Humidifier Settings
+
+VeSync humidifiers have settings other than just a target humidity, like a manual mist level and an on/off display toggle.
+These are exposed as configuration entities alongside the humidifier itself.
+
+| Entity ID                                 | Description                                    |
+| ----------------------------------------- | ---------------------------------------------- |
+| `number.<humidifier name>_mist_level`     | The target mist level of the humidifier        |
+| `light.<humidifier name>_night_light`     | The brightness of the humidifier's night light |
+| `switch.<humidifier name>_display`        | The humidifier's display                       |
+| `switch.<humidifier name>_automatic_stop` | The humidifier's automatic stop feature        |
```

Comment:
This integration uses config entries, and entity IDs are not guaranteed and thus should not be mentioned like this. 

Instead, name and describe the sensor it creates.